sources/packages/web/src/views/aest/AESTDashboard.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <v-container>
+  <full-page-container>
     <formio formName="aestdashboard"></formio>
```

Comment:
please use `formio-container`

Reply:
is the dashborad changes part of the next PR?

Reply:
yes